packages/web-app-files/tests/unit/views/views.setup.js

```diff
@@ -207,20 +205,6 @@ export const getStore = function ({
               closed: sidebarClosed
             },
             namespaced: true
-          },
-          pagination: {
```

Comment:
fyi: there's no pagination store module in the files app for quite some time now. Removed the dead code here in the tests as well now.